traffic_portal/app/src/common/api/TenantService.js

```diff
@@ -17,53 +17,67 @@
  * under the License.
  */
 
-var TenantService = function(Restangular, messageModel) {
+var TenantService = function($http, ENV, messageModel) {
 
     this.getTenants = function(queryParams) {
-        return Restangular.all('tenants').getList(queryParams);
+        return $http.get(ENV.api['root'] + 'tenants', {params: queryParams}).then(
+            function (result) {
+                return result.data.response;
+            },
+            function (err) {
+                console.error(err);
```

Comment:
did you want to make these "throw" the error? you want to check all your PRs to make sure you did that if that was your intent?

Reply:
uh oh, looks like my `sed` didn't take. I'll probably look at that over the weekend. It's a lot to do when I'm supposed to be working on other things :P